.github/workflows/CI.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: mrtravis/gridtools:gcc-9-mpi
+    strategy:
+      build_type: [Debug]
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Configure
+        run: |
+          mkdir -p build && cd build
+          cmake .. -DGHEX_BUILD_TESTS=ON
```

Comment:
No hint to GridTools path is required since everything is already set in `mrtravis/gridtools:gcc-9-mpi`, right?

Reply:
That's my hope :)
I will try